mm-bot/src/services/starknet.py

```diff
@@ -106,12 +107,15 @@ async def get_order_events(from_block_number, to_block_number) -> list[SetOrderE
 
 
 async def create_set_order_event(event):
-    is_used = await get_is_used_order(event.data[0])
+    order_id = event.data[0]
+    recipient_address = hex(event.data[2])
+    amount = event.data[3]
```

Comment:
Remember that amount is actually a u256 therefore it should be data[3] and data[4] probably

Reply:
Yes, it is the original implementation, can we discuse it? Specially how to parse these two elements into one int

Reply:
100%